util/http.go

```diff
@@ -111,15 +111,15 @@ func ExtractAuth(c echo.Context) (string, error) {
 		return "", &HttpError{
 			Code:    http.StatusUnauthorized,
 			Reason:  ERR_INVALID_AUTH,
-			Details: "invalid api key was specified",
+			Details: "Unsupported authorization scheme: Bearer is a required prefix, also make sure you have a valid and unexpired API key. An API key is required to use this endpoint.",
 		}
 	}
 
 	if parts[0] != "Bearer" {
```

Comment:
can we put this check before the one above and write the condition as `if len(parts) > 0 & parts[0] != "Bearer"`? so they get the ERR_AUTH_MISSING_BEARER reason when they don't provide bearer, instead of ERR_INVALID_AUTH. then you can make the language more targeted and say "Bearer prefix is required but missing" for that case